src/Components/Slides/SlidesForm.js

```diff
@@ -1,32 +1,107 @@
-import React, { useState } from 'react';
-import '../FormStyles.css';
+import axios from "axios";
+import React, { useState } from "react";
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import "../FormStyles.css";
+import Swal from 'sweetalert2'
 
-const SlidesForm = () => {
-    const [initialValues, setInitialValues] = useState({
-        name: '',
-        description: ''
-    });
-
-    const handleChange = (e) => {
-        if(e.target.name === 'name'){
-            setInitialValues({...initialValues, name: e.target.value})
-        } if(e.target.name === 'description'){
-            setInitialValues({...initialValues, description: e.target.value})
+const SlidesForm = ({slides}) => {
```

Comment:
it receive only one slide or a list of slides? if is only one slide the parameter should be slide, if its a list of slides I believe that slides.name will never work.

Reply:
done